File: app/src/main/java/org/kindone/willingtodo/recyclerlist/RecyclerListItem.java
package org.kindone.willingtodo.recyclerlist;

/**
 * Created by kindone on 2016. 12. 18..
 */

public interface RecyclerListItem {
    long getId();
    String getTitle();

    public interface Updater {
        RecyclerListItem update(RecyclerListItem item);
    }
}

File: app/src/main/java/org/kindone/willingtodo/recyclerlist/RecyclerListItem.kt
package org.kindone.willingtodo.recyclerlist

/**
 * Created by kindone on 2016. 12. 18..
 */

interface RecyclerListItem {
    fun getId(): Long
    fun getTitle(): String

    interface Updater {
        fun update(item: RecyclerListItem): RecyclerListItem
    }
}
